mrp_bom_location/models/mrp_bom.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MrpBom(models.Model):
+    _inherit = "mrp.bom"
+
+    location_id = fields.Many2one(
```

Comment:
If these fields are related, shouldn't these be readonly here ?

Reply:
Yes again :)

Reply:
I see confusing for the user to use this related field as location. Maybe it's better to decouple from the picking type? And due to picking type is not used for any other thing. What about if you want to use a location that hasn't got any picking type related to?

Reply:
After doing some tests I tend to agree with @pedrobaeza that it's confusing.

Especially with such an help message.

![image](https://user-images.githubusercontent.com/778828/37473971-95949924-286f-11e8-8ce5-99da360faef1.png)